api-reference/10 UI Components/dxDropDownList/1 Configuration/onContentReady.md

```diff
@@ -1,5 +1,3 @@
----
-##### merge
+If you use a [DataSource](/api-reference/30%20Data%20Layer/DataSource '/Documentation/ApiReference/Data_Layer/DataSource/') instance to specify data for a UI component, the **contentReady** event fires each time the [load()](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/load().md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#load') method of the DataSource instance is called as well as when UI component content is ready. In this case, when you create the UI component, the **contentReady** event is raised twice: when the UI component's content is ready (at this moment, the UI component does not display items) and when the DataSource is loaded (the UI component displays the loaded items).
 
----
-If data displayed by the UI component is specified using a [DataSource](/api-reference/30%20Data%20Layer/DataSource '/Documentation/ApiReference/Data_Layer/DataSource/') instance, the **contentReady** event fires each time the [load()](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/load().md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#load') method of the DataSource instance is called as well as when UI component content is ready. In this case, when you create the UI component, the **contentReady** event is raised twice: when the UI component's content is ready (at this moment, the UI component does not display items) and when the DataSource is loaded (the UI component displays the loaded items).
\ No newline at end of file
+The **contentReady** event also fires when the component content changes or the [repaint()](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Methods/#repaint) method is called.
```

Comment:
Lists should work great in this case. Let's make two of them: one containing cases regardless of the DataSource usage, the other containing cases when the DataSource is used.